From: Paul Boddie <paul@boddie.org.uk>
Date: Thu, 21 Sep 2023 18:04:26 +0200
Subject: Employ Font Awesome 4.x stylesheet naming.

Since the Debian package for Font Awesome appears to provide files
corresponding to the 4.x series of Font Awesome, the template needs
to refer to font-awesome.css instead of all.css in the installed
css directory.
---
 src/moin/templates/base.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/moin/templates/base.html b/src/moin/templates/base.html
index f41e455..0848605 100644
--- a/src/moin/templates/base.html
+++ b/src/moin/templates/base.html
@@ -58,7 +58,7 @@
                         <link media="all" rel="stylesheet" href="{{ theme_static('css/variables.css') }}" />
                     {%- endif %}
                     <link media="all" rel="stylesheet" href="{{ url_for('serve.files', name='pygments', filename='css/colorful.css') }}" />
-                    <link media="all" rel="stylesheet" href="{{ url_for('serve.files', name='font_awesome', filename='css/all.css') }}" />
+                    <link media="all" rel="stylesheet" href="{{ url_for('serve.files', name='font_awesome', filename='css/font-awesome.css') }}" />
                     <link media="all" rel="stylesheet" href="{{ url_for('static', filename='css/common.css') }}" />
                     <link media="all" rel="stylesheet" href="{{ theme_static('css/theme.css') }}" />
                     {{ stylesheets }}
